app.js

```diff
@@ -133,6 +134,14 @@
         .setAttribute('data-clipboard-text', webfontCss);
   }
 
+  function formatCSS(originalString, selector) {
```

Comment:
copied css was a little sloppy before. also nicer to have a single place to edit formatting.